README.md

```diff
@@ -150,6 +150,33 @@ yab -t ~/keyvalue.thrift -P ~/hosts.json keyvalue KeyValue::get -r '{"key": "hel
 ```bash
 yab -t ~/keyvalue.thrift -p "http://localhost:8080/rpc" keyvalue KeyValue::get -r '{"key": "hello"}'
 ```
+#### Proto
+The following examples assume that the Proto service running looks like:
+```proto
+service KeyValue {
+  rpc GetValue(Request) returns (Response) {}  
+}
+```
+
+If a gRPC service was running with name `KeyValue` on `localhost:12345` with proto package name `pkg.keyvalue` and 
```

Comment:
we also should mention that if reflection is enabled on the server, there's no need to specify the binary file

Reply:
Good catch! added a note.